### PR TITLE
Scrolling: Boundary feedback

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1583
[Problem] Boundary feedback needs implementation
[Solution]
     - added examples in UIComponents

[Screenshot]
image

    
Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/scrollview.less [hunk 1]
```diff
@@ -94,22 +94,24 @@
 
 .ui-overflow-indicator-top,
 .ui-overflow-indicator-bottom {
-	position: absolute;
+	position: fixed;
 	display: none;
-	width: 100%;
-	height: 29 * @unit_base;
+	width: 10%;
+    transform: scaleX(10);
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/scrollview.less:100]: Fix indent

File: src/css/profile/mobile/common/scrollview.less [hunk 2]
```diff
@@ -94,22 +94,24 @@
...
+    transform-origin: left;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/scrollview.less]: Fix indent

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/scrollview.less]: Done.

Thank you for review. I think I fixed all issues found by you.

File: src/js/profile/mobile/widget/Scrollview.js [hunk 3]
```diff
@@ -215,7 +215,65 @@
...
+					clipOffset = DOMUtils.getElementOffset(clip),
+					viewHeight = DOMUtils.getElementHeight(view),
+					viewWidth = DOMUtils.getElementWidth(view);
+
+				self._clearIndicator(clip);
+
+				switch (self.options.scroll) {
+					case "x":
+					case "xy":
+						// @todo
```
Comment (DohyungLim) [inline src/js/profile/mobile/widget/Scrollview.js]: Is it planned to be implemented in the next patch?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: src/js/profile/mobile/widget/Scrollview.js (hunk 3 above)
Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/Scrollview.js]: Thank you for review :) . This is good question. This is copy of method from `core/widget/Scrollview.js`. I didn't plan yet when this feature will be implemented. I'll remove `@todo` comment.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**